core-modules/generic-core-terms-04-our-expectations.md.md

```diff
@@ -0,0 +1,13 @@
+## Our Expectations
+
+In participating in our vulnerability disclosure program in good faith, we ask that you:
+
+- Play by the rules, including following this policy and any other relevant agreements. If there is any inconsistency between this policy and any other applicable terms, the terms of this policy will prevail;
+- Report any vulnerability you’ve discovered promptly;
+- Avoid violating the privacy of others, disrupting our systems, destroying data, and/or harming user experience;
+- Use only the Official Channels to discuss vulnerability information with us;
+- Handle the confidentiality of details of any discovered vulnerabilities according to our Disclosure Policy;
+- Perform testing only on in-scope systems, and respect systems and activities which are out-of-scope;
+- If a vulnerability provides unintended access to data: Limit the amount of data you access to the minimum required for effectively demonstrating a Proof of Concept; and cease testing and submit a report immediately if you encounter any user data during testing, such as Personally Identifiable Information (PII), Personal Healthcare Information (PHI), credit card data, or proprietary information;
+- You should only interact with test accounts you own or with explicit permission from the account holder; and
```

Comment:
What about in scenarios in which the holder is unclear, or potentially is not the party which can/should be capable of granting such access? (e.g. employee providing a company account to a researcher)

Reply:
Mmm... That's somewhat of an edge case. I'd think that's more of an operational consideration than one that could be covered by the policy